sqs/src/main/kotlin/com/playkids/onboarding/sqs/SQSEmitter.kt

```diff
@@ -0,0 +1,8 @@
+package com.playkids.onboarding.sqs
+
+import kotlinx.coroutines.Job
+
+interface SQSEmitter{
```

Comment:
SQS is a way to implement something. As we are SQS users (not developers), our interfaces should not know about SQS.